app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -138,6 +144,184 @@ fun TestFlipCard(cardFace: CardFace, onClickToList: () -> Unit, id: String) {
+
+        Image(painter = cardImage, contentDescription = null, modifier = Modifier.fillMaxSize())
+        Column(
+            modifier = Modifier.padding(horizontal = 10.dp, vertical = 15.dp)
+        ) {
+            Topic()
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .weight(1f)
+                    .background(Color.White)
+            )
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(1.dp)
+                    .background(Gray03)
+            )
+            Starter()
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(1.dp)
+                    .background(Gray03)
+            )
+            ShareBottom()
+        }
+    }
+}
+
+@Composable
+fun Topic() {
+    Column(
+        modifier = Modifier
+            .padding(24.dp)
+    ) {
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Text(
+                modifier = Modifier.align(Alignment.CenterStart),
+                text = stringResource(R.string.detail_topic),
+                color = Gray05,
+            )
+            Icon(
+                modifier = Modifier
+                    .align(Alignment.CenterEnd)
+                    .size(24.dp)
+                    .padding(2.dp)
+                    .clickable {  }
+                ,
+                painter = painterResource(id = R.drawable.ic_star_empty),
+                tint = Gray05,
+                contentDescription = null
+            )
+        }
+        Spacer(
+            modifier = Modifier
+                .height(8.dp)
+                .fillMaxWidth(),
+        )
+        Text(
+            text = "천사처럼 자고 있던 내 연인, 슬며시 다가가 머리를 쓰다듬는데 나도 몰랐던 가발이 벗겨졌다. 모르는척 다시 씌워준다 vs 왜 날 속였냐고 따진다.",
+            style = TalkbbokkiTypography.b2_bold
+        )
+    }
+}
+
+@Composable
+fun Starter() {
+    Column(
+        modifier = Modifier
+            .height(114.dp)
+            .padding(24.dp)
+    ) {
+        Row(modifier = Modifier.height(24.dp)) {
+            Text(
+                text = stringResource(R.string.detail_starter),
+                color = Gray05,
+            )
+            Spacer(
+                modifier = Modifier
+                    .width(4.dp)
+                    .fillMaxHeight()
+            )
+            Icon(
+                modifier = Modifier
+                    .size(18.dp)
+                    .padding(2.dp)
+                    .clickable { },
+                painter = painterResource(id = R.drawable.ic_refresh),
+                tint = Gray05,
+                contentDescription = null
+            )
+        }
+        Spacer(
+            modifier = Modifier
+                .height(8.dp)
+                .fillMaxWidth()
+        )
+        Text(
+            text = "1인 1닭 가능할 것 같은 사람", style = TalkbbokkiTypography.b2_bold
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)